src/catalog/src/error.rs

```diff
@@ -57,6 +57,29 @@ pub enum Error {
         source: BoxedError,
     },
 
+    #[snafu(display("Failed to list flows in catalog {catalog}: {source}"))]
+    ListFlows {
+        #[snafu(implicit)]
+        location: Location,
+        catalog: String,
+        source: BoxedError,
+    },
+
+    #[snafu(display("Flow info not found: {flow_name} in catalog {catalog_name}"))]
+    FlowInfoNotFound {
+        flow_name: String,
+        catalog_name: String,
+        #[snafu(implicit)]
+        location: Location,
+    },
+
+    #[snafu(display("Can't convert value to json: {raw:?}"))]
+    Json {
+        raw: serde_json::error::Error,
+        #[snafu(implicit)]
+        location: Location,
+    },
```

Comment:
How about
 ```suggestion
    #[snafu(display("Failed to serialize to json: {}", input))]
    SerializeToJson {
        input: String,
        #[snafu(source)]
        error: serde_json::error::Error,
        #[snafu(implicit)]
        location: Location,
    },
```

Reply:
> How about

serde_json::error::Error didn't impl our `ErrorExt` so I don't think that is viable?

Reply:
Yes, that why we need to add ` #[snafu(source)]`.